src/screens/SurveyDetailScreen.tsx

```diff
@@ -27,17 +27,27 @@ type Props = {
   route: RouteProp<any, any>;
 };
 
-const DateComponent = ({ start, date }: { start?: boolean; date: string }) => {
+const DateComponent = ({
+  start,
+  date,
+  isMultilingual
+}: {
+  start?: boolean;
+  date: string;
+  isMultilingual?: boolean;
+}) => {
   return (
     <Wrapper style={styles.noPaddingBottom}>
       <WrapperRow center spaceBetween>
         <View>
           <RegularText small>
             {start ? texts.survey.dateStart.de : texts.survey.dateEnd.de}
           </RegularText>
-          <RegularText italic small>
-            {start ? texts.survey.dateStart.pl : texts.survey.dateEnd.pl}
-          </RegularText>
+          {!!isMultilingual && (
```

Comment:
new prop isn't passed yet to `DateComponent`, correct?

Reply:
correct. added it.